src/main/webapp/query-control.js

```diff
@@ -289,6 +295,16 @@ function replaceValueIfEmpty(dataListId) {
   }
 }
 
+function showEditTitle() {
```

Comment:
Maybe 'edit title' button could be hidden when looking at geoJson, since that doesn't have a title. Though it may also be useful to edit tile there and see it when you switch back to amcharts

Reply:
Ah, that's a good point! I think it would be more confusing to users to see the button when there isn't a title, so I'll hide it.